sdks/python/src/opik/evaluation/rest_operations.py

```diff
@@ -95,3 +96,28 @@ def log_test_result_feedback_scores(
     client.log_traces_feedback_scores(
         scores=all_trace_scores, project_name=project_name
     )
+
+
+def log_experiment_scores(
```

Comment:
I'd suggest moving it to `opik.api_objects.Experiment` class and call it directly from there without putting a new function to rest_operations helper module (it might also be useful for other users).